cmd/sharded-test-server/virtual.go

```diff
@@ -58,6 +59,7 @@ func startVirtual(ctx context.Context, index int, logDirPath string) (<-chan err
 		"--requestheader-group-headers=X-Remote-Group",
 		fmt.Sprintf("--secure-port=%d", 7444+index),
 	)
+	fmt.Fprintf(out, "running: %v\n", strings.Join(commandLine, " ")) // nolint: errcheck
```

Comment:
Thanks! Could you please edit line 38 (`logger.Info("starting virtual-workspaces standalone server"...`) instead?

Reply:
Or keep this here, and remove 38?

Reply:
Removed both line and unused logger instance. @ncdc

Reply:
Thanks!